src/feasibility_checker.jl

```diff
@@ -0,0 +1,110 @@
+const MOD = MathOptSetDistances
+const MOIU = MOI.Utilities
+
+variable_primal(model) = x -> MOI.get(model, MOI.VariablePrimal(), x)
+variable_primal_start(model) = x -> MOI.get(model, MOI.VariablePrimalStart(), x)
+
+mutable struct FeasibilityCheckerOptions
+    names::Bool
+    index::Bool
+    function FeasibilityCheckerOptions(;names = true, index = true)
+        return new(
+            names,
+            index,
+        )
+    end
+end
+
+function constraint_violation_report(model::MOI.ModelLike;
+    varval::Function = variable_primal(model),
+    distance_map::Dict = Dict(),
+    options = FeasibilityCheckerOptions())
+
+    largest, vec = constraint_violation(model, varval = varval, distance_map = distance_map)
+
+    str = " Feasibility report\n\n"
```

Comment:
maybe these using a multiline string:
s = """Feasibility report

...
"""

Reply:
bump

Reply:
Agree! I will be back soon to it